docs-devsite/firestore_lite.querycompositefilterconstraint.md

```diff
@@ -0,0 +1,35 @@
+Project: /docs/reference/js/_project.yaml
```

Comment:
Did I forget to press a button earlier today . . . ??

Anyway, my question was: why are these output files included in the PR? Generally we do those in a follow-on CL.  Thanks!

Reply:
@egilmorez ... wrt to the feature, this content looks fine. Do you want resolution to this discussion before I give the single approval that will unblock?

Reply:
Nope, go ahead Mark, thanks!  I'm curious see if a doc CL crosses my "desk."